abb_irb4400_support/package.xml

```diff
@@ -32,9 +33,11 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <!-- Typical run depends, not yet populated
-  <run_depend>abb_assets</run_depend>
+  <build_depend>roslaunch</build_depend>
```

Comment:
If you add the `DEPENDENCIES` arg (see earlier comment), we should state the `roslaunch` version here, just to be safe. See [here](https://github.com/ros-industrial/fanuc/blob/5e2cf28d4b65af0724566a132c0f56aff6f737d2/fanuc_m10ia_support/package.xml#L36) for an example.